code/modules/client/client procs.dm

```diff
@@ -135,36 +164,43 @@
 	prefs.last_ip = address				//these are gonna be used for banning
 	prefs.last_id = computer_id			//these are gonna be used for banning
 
+
 	. = ..()	//calls mob.Login()
 	chatOutput.start() // Starts the chat
 
+	if(byond_version >= 511)
```

Comment:
sorry should have been more clear
swap this to early returns.

`if(byond_version < 512)` too old qdel/return